.gitignore

```diff
@@ -190,4 +190,7 @@ tests/FSharpLint.FunctionalTest.TestedProject/FSharpLintMSBuildTaskTest/
 
 DesignTimeBuild/
 .vs/
-.ionide/
\ No newline at end of file
+.ionide/
+
+# BenchmarkDotNet Results
+/tests/FSharpLint.Benchmarks/BenchmarkDotNet.Artifacts
```

Comment:
@Numpsy this being a relative path... shouldn't it **not** start with a slash? otherwise it looks like an absolute path

Reply:
Hmm, added it via the Visual Studio UI and just took what it did. Updated though.

(fwiw, the ```/paket-files/``` ignore a few lines earlier is the same